apps/dashboard/src/components/workflow-editor/test-workflow/snippet-editor.tsx

```diff
@@ -1,27 +1,28 @@
-import { Editor } from '@monaco-editor/react';
+import { useMemo } from 'react';
```

Comment:
in the Workflow Editor --> Trigger tab use the CodeMirror editor instead of Monaco